openshift/e2e-common.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
-export EVENTING_NAMESPACE=knative-eventing
+export EVENTING_NAMESPACE="${EVENTING_NAMESPACE:-knative-eventing}"
 export SYSTEM_NAMESPACE=$EVENTING_NAMESPACE
-export OLM_NAMESPACE=openshift-marketplace
+export ZIPKIN_NAMESPACE=$EVENTING_NAMESPACE
```

Comment:
don't we set that to zipkin ? or that's cause you wanna deploy zipkin within eventing ns during tests?

Reply:
@devguyio nope, see here: https://github.com/knative/eventing/commit/4c422f11512f66e4382b66d5df00ac3889cccb95#diff-eb024bb2073b22fd3bbb566bb0493f364179f68804a69c62ca28426fa6cd9511R22